### ISSUE TITLE
GitHub: Unable to render rich display

### ISSUE THREAD

**Issue by Vangelis66** [martok/palefill#70]
**Browser**: Serpent 52.9.0/**UXP** (32-bit)
Extension version: palefill-**1.25**

STR: 
1. Load https://github.com/ossrs/srs-cloud#architecture
2. **Expected** result (captured with a Chromium **86** fork): 

![mermaid1](https://user-images.githubusercontent.com/9669492/209240676-8d2b8a75-d020-4dae-9123-061236f6a2d4.png)

and, further down: 

![mermaid1a](https://user-images.githubusercontent.com/9669492/209241510-16628c29-5b0d-4a3f-8e1f-68bac8b84b5d.png)

3. Actual result: 

![mermaid2](https://user-images.githubusercontent.com/9669492/209241005-8395023b-0d3b-4c0f-af4d-76e34159eb3f.png)

and, further down: 

![mermaid2a](https://user-images.githubusercontent.com/9669492/209241816-70923ee5-811e-41bd-8251-35872aa6d5c4.png)

In text form:

>**Unable to render rich display**
>
>this.el.replaceChildren is not a function
>
>For more information, see 
>[https://docs.github.com/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams](https://docs.github.com/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams)

>Unable to render rich display
>
>undefined
>
>For more information, see 
>[https://docs.github.com/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams](https://docs.github.com/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams)

The so-called "**mermaid diagrams**" are served from: 
```
https://mermaid.live/edit#pako:eNptUk1vwjAM_StRDhNIFHbYLmxCQhSEtC_UMi4th7QxbUeTVKnLQIj_viSdthXtUNt57znPsXqmqeJAx3RXqs80ZxrJc_AQS0JkVsgj8bwJEZnAng2PiR5NkhORpuWj7jtZ3SSZZlVOwiB0ALFVEJlA7oa3riVElpSw_aV9R99_0z4coFRVy4PkNlk7Z26FNi-V2tctghqYmDaYkxvibwITN8p3zfZkJbO3dpafWxYLUUH2F_NIVTLcKS0c2tp4_wgn5L0yL-Tg6gBKYDV0BTPQmKi2Xq7XqyvzNcgUJM5K1fBWP32JzDcyY47M6NuufKWVAMyhqd1Er2bZ82OlNILuCn2V7q-xuTxEvSHIg9tsqoEb54KV9RCP2L9yCoAXddRziTxtDE0HVIAWrODmlzhbcUzNKAJiOjYlZ3of01hejI41qMKTTOkYdQMD2lScIfgFM8sSXXDOC1S6xS5fkK3EMQ
```
Attempting to load **directly that URI** in a **UXP**-based browser will produce a blank tab 😢 , because of two well-known **UXP** "arch-enemies": 
1. **Regex Name Capturing Groups**
`SyntaxError: invalid regexp group`
2. **dynamic module imports**
`SyntaxError: expected expression, got keyword 'import'`

... As I recently wrote elsewhere, **UXP** lies on its deathbed 😞 ...

**Comment by martok** [comment]
> 1. **Regex Name Capturing Groups**
>        `SyntaxError: invalid regexp group`

Not to spoil the surprise, but... :wink:

**Comment by Vangelis66** [comment]
I can confirm the issue is **fixed** with `palefill-v1.25-4-git-20221228-g9674444` code snapshot installed [in **Serpent 52.9.0** (32-bit)]; many thanks indeed! 👍 🎉 

> Not to spoil the surprise, but... 😉

Nothing "spoiled" here 😜 ; 
https://github.com/roytam1/UXP/compare/4db6fa8...81a8ac4
have landed in last weekend's builds already! 
Your stupendous work has been **highly appreciated**! Congrats 🥇
